docs/tracing.md

```diff
@@ -26,11 +26,12 @@ identifying elongated function calls.
 
 Firecracker implements instrumentation based tracing via
 [`log`](https://github.com/rust-lang/log) and
-[`log_instrument`](https://github.com/JonathanWoollett-Light/log-instrument),
-outputting a `Trace` level log when entering and exiting every function.
+[`log_instrument`](../src/log-instrument), outputting a `Trace` level log when
+entering and exiting every function.
 
-It is disabled by default at compile-time. Tracing functionality has no impact
-on the release binary.
+Enabling tracing impact Firecracker binary size and its performance, so it is
```

Comment:
I don't think this is correct. Enabling/disabling tracing is a runtime action and can't change FC binary size. Did you mean adding instrumentation? It does indeed change the binary size.
However I'm not quite sure about the original "It is disabled by default at compile-time" either. Maybe it can be reworded as "tracing is disabled by default"?

Reply:
I've posted a suggestion comment https://github.com/firecracker-microvm/firecracker/pull/4423/commits/ded99cf417323fb18222b704acea6b7cc6a1dc25#r1478237955 (GitHub doesn't allow them in comment chains it seems)